nixos/modules/services/web-apps/photoprism.nix

```diff
@@ -104,6 +104,7 @@ in
         StateDirectory = "photoprism";
         WorkingDirectory = "/var/lib/photoprism";
         RuntimeDirectory = "photoprism";
+        ReadWritePaths = [ cfg.originalsPath cfg.importPath cfg.storagePath ];
```

Comment:
`cfg.importPath` doesn’t necessarily work here as it may be just a relative path.